torchvision/datapoints/_bounding_box.py

```diff
@@ -44,6 +50,10 @@ class BoundingBoxes(Datapoint):
 
     @classmethod
     def _wrap(cls, tensor: torch.Tensor, *, format: BoundingBoxFormat, canvas_size: Tuple[int, int]) -> BoundingBoxes:
+        if tensor.ndim == 1:
+            tensor = tensor.unsqueeze(0)
+        elif tensor.ndim != 2:
+            raise ValueError(f"Expected a 1D or 2D tensor, got {tensor.ndim}D")
```

Comment:
I don't have a super strong opinion on that and I don't mind removing it, but IMO passing other number of dims indicates a user mistake upstream.

We don't support batches of bounding boxes (i.e. we can't batch BoundingBoxes objects where `num_bboxes` differs per object) so I'm not sure we should pretend that we do.

Reply:
Due to this, the following check is obsolete:

https://github.com/pytorch/vision/blob/9b82df43341a6891f652be1803abd1d1d05bfbb2/torchvision/transforms/v2/_misc.py#L392-L393

I've removed it and the (now failing) test for it in b962a4b2c4475f3d9a57e1e134ad77bada42af86. If we decide to not go with this, make sure to revert the commit as well.